config_tors.yml

```diff
@@ -15,9 +12,14 @@
       with_items:
       - "{{ config_dir }}/.aws"
       - "{{ config_dir }}/.evalai"
-  
+    
     - name: aws config
-      shell: 'echo "[default] \nregion=us-east-1\naws_access_key_id = {{ aws_access_key_id }}\naws_secret_access_key = {{ aws_secret_access_key }}" > {{config_dir }}/.aws/config'
-
+      template:
```

Comment:
The main reason I opened this PR: if you want to make the creation of files a little neater, I would suggest this approach. Using `jinja2` allows you to add variables inside the files.

Reply:
That's really nice to know, thanks!